### ISSUE TITLE
Dlc game won't load

### ISSUE THREAD

**Issue by maybilll** [sudachi-emu/sudachi#94]
Dlc Game like ac ezio collection or ff x 1/2 hd wont load on androis version

**Comment by tahirjamil** [comment]
Mine mortal kombat 1 and super smash bros doesn't load too, first when i install it loads but after i shutdown or restart the emulator the update and dlc shows in settings but the game doesn't recognise update and dlc

**Comment by DZXTRICKS** [comment]
A lot of DLC doesn't work work for me. I think some anti-emulation measures are in place (just an assumption, I know nothing about this matter) when our game updated with DLC? Or maybe the DLC needed some sort of online verification. Either way, Snowrunner and Ori and the blind forest doesn't work with DLC installed

**Comment by zzzribas** [comment]
Yes, im also getting this problem, if i install manually it works until i close the emulator and open again, i changed my directory of nand can it be it? Does it check when it starts for the default nand and after that unloads the addons or dlcs?
